webserver/index.html

```diff
@@ -34,6 +35,15 @@ <h1><center>ETA: <div class="ETA">None</div> </center></h1>
                         document.getElementsByClassName("place")[0].innerHTML = response.place;
                         document.title = "2Bored2Wait - " + response.place;
                         document.getElementsByClassName("ETA")[0].innerHTML = response.ETA;
+                        if (response.finTime != "Never") {
+                            const finishedDate = new Date(response.finTime);
+                            let hours = finishedDate.getHours();
+                            let minutes = finishedDate.getMinutes();
+                            if (hours < 10) hours = `0${hours}`;
+                            if (minutes < 10) minutes = `0${minutes}`;
+                            document.getElementsByClassName("finishedTime")[0].innerHTML = `${hours}:${minutes}`;
```

Comment:
```suggestion
                            const date = (new Date(response.finTime)).toLocaleTimeString();
                            document.getElementsByClassName("finishedTime")[0].innerHTML = date;
```
This would probably be easier and also works for 12hour systems